PD-1624 / 25.04 / Pd 1624 change system dataset needs additional info (#3382)
truenas/documentation: content/CORETutorials/SystemConfiguration/SettingtheSystemDataset.md

You cannot move the system dataset to a passphrase-encrypted or read-only pool.

> When moving the system dataset, TrueNAS replicates the data to a new dataset and sets it as active. TrueNAS does not delete the old dataset in case you moved the system dataset temporarily or need to recover the data from it.

Reviewer: ```suggestion
When moving the system dataset, TrueNAS replicates the existing dataset and activates the new dataset. TrueNAS does not delete the old dataset in case you moved the system dataset temporarily or need to recover the data from it.
```
